cadquery/selectors.py

```diff
@@ -300,6 +300,45 @@ def filter(self, objectList):
         return r
 
 
+class RadiusSelector(TypeSelector):
```

Comment:
I'd consider renaming to RadiusNthSelector , but I'm open for discussion.

Reply:
I agree, that's will be more consistent with the other selectors.